io_scene_niftools/modules/nif_export/geometry/mesh/__init__.py

```diff
@@ -446,7 +469,7 @@ def export_tri_shapes(self, b_obj, n_parent, trishape_name=None):
                             triangles=trilist,
                             trianglepartmap=bodypartfacemap,
                             maximize_bone_sharing=(bpy.context.scene.niftools_scene.game in ('FALLOUT_3', 'SKYRIM')),
-                            part_sort_order = part_order)
+                            part_sort_order=part_order)
```

Comment:
Presume since the others have no spaces this is right.

Reply:
If not, pycodestyle complains.